zask/ext/zerorpc/__init__.py

```diff
@@ -296,7 +296,7 @@ def server_inspect_exception(self, request_event, reply_event, task_context, exc
             'referrer': '-',
             'user_agent': '-',
             'cookies': '-',
-            'request_time': _milli_time() - start,
+            'request_time': _milli_time() - start if start else 0,
```

Comment:
I tried that earlier, but if we put `-` it breaks the logging module because that expects an integer to format into a the log string with `%d`.